docs/how-to/troubleshooting-omnitrace-linux.rst

```diff
@@ -0,0 +1,68 @@
+.. meta::
+   :description: Omnitrace documentation and reference
+   :keywords: Omnitrace, ROCm, profiler, tracking, visualization, tool, Instinct, accelerator, AMD
+
+****************************************************
+Troubleshooting Omnitrace on Linux
```

Comment:
why on Linux?
Thought it only ran on Linux

By the way, this could be a post-installation step instead of a separate topic. 
If you maintain as a separate topic, you should be clear that it's only for users running RH.

Perhaps title something like: 
Troubleshooting OT on RHEL

Reply:
I reworked the troubleshooting page a bit, moving the RHEL instrumentation issue down to a H2 (making it clear that this only applies to RHEL), at the same heading level as the RPATH issues, which applies to all distributions.

I also removed the reference to Linux, head and in the index/ToC. You're right; it's redundant.